fast_llm/models/gpt/config.py

```diff
@@ -71,6 +71,17 @@ class DiffusionLlamaGPTHuggingfaceCheckpointFormat(GPTHuggingfaceCheckpointForma
     trust_remote_code: typing.ClassVar[bool] = True
 
 
+class LlavaGPTHuggingfaceCheckpointFormat(GPTHuggingfaceCheckpointFormat):
```

Comment:
I don't understand, is all that added complexity just so we can call it `llava` instead of either `pixtral` or `mistral`?